corehq/apps/fixtures/upload/run_upload.py

```diff
@@ -189,4 +190,22 @@ def _update_progress(table_count, item_count, items_in_table):
                         old_data_item.add_location(location_cache.location,
                                                    transaction=transaction)
 
+    clear_fixture_quickcache(data_types)
     return return_val
+
+
+def clear_fixture_quickcache(data_types):
+    """
+    Clears quickcache for fixtures.dbaccessors
+
+    Args:
+        List of FixtureDataType objects in a domain
+    """
+    if not data_types:
+        return
+    type_ids = []
+    for data_type in data_types:
+        data_type.clear_caches()
+        type_ids.append(data_type.get_id)
+    from corehq.apps.fixtures.dbaccessors import get_fixture_items_for_data_types
+    get_fixture_items_for_data_types.clear(data_types[0].domain, type_ids)
```

Comment:
I thought about it, but this would really never be called with a mix of domains. Even if it's called, there is no effect in that the cache key won't exist and nothing happens.